### DISCUSSION TITLE
2nd y-axis of a chart

### DISCUSSION THREAD

**Discussion by bobdguan** [jshinonome/vscode-q#90] [General]
Hi Jo, for line chart in query grid, is there a way to have a 2nd y-axis, so some series are plotted on the 1st and some others of difference scales are plotted on the 2nd? I played a bit but didn't find it, if not available, is that something you could add to the extension? Thanks!

Btw, great extension and I see a lot more usage in my workplace now. 

Bob

**Comment by jshinonome** [comment]
Hi Bob, for the line chart in the query grid, no way to add a 2nd y-axis, but in visualization mode, you can have the 2nd y-axis by click the series name as below.
![image](https://github.com/jshinonome/vscode-q/assets/64458591/98cd2817-4aaa-4241-9547-16cb0d414cb6)

**Comment by bobdguan** [comment]
Nice, thanks!

**Comment by bobdguan** [comment]
Actually a follow up one: Y Line above doesn't specify an x-axis, but can do 2nd y-axis, while XY Line, I can have an x-axis, but cannot do 2nd y-axis, is there a plot that i can have both x-axis, y-axis and a secondary y-axis for a line chart?

**Reply by jshinonome** [reply]
Still, select "Y-line", there is a "Group By" next to "Y-line", drag and drop your x-axis column there. If "Group By" doesn't work, give a try "Order By" etc on the same line.

**Reply by bobdguan** [reply]
group by works, thanks!
